src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
Could you clarify why not using the term commonly used by Selenium is not a good idea here?